react-state/my-app/src/ToggleButton.jsx

```diff
@@ -0,0 +1,27 @@
+import { useState } from 'react';
+import './ToggleButton.css';
+
+export default function ToggleButton({ text, color }) {
+  let [isClicked, setIsClicked] = useState(false);
+  console.log('useState:', isClicked);
+
+  function handleClick() {
+    console.log('before setter:', isClicked);
+    isClicked ? setIsClicked(false) : setIsClicked(true);
+    console.log('after setter:', isClicked);
+  }
+
+  if (isClicked) {
+    return (
+      <button onClick={handleClick} className="white">
+        {text}
+      </button>
+    );
+  } else {
+    return (
+      <button onClick={handleClick} className={color}>
+        {text}
+      </button>
+    );
+  }
```

Comment:
We can simplify this. Since the only thing that changes between these two elements is the `className`, it's better to move the ternary there.

```
return (
      <button onClick={handleClick} className={ isClicked ? 'white' : color }>
        {text}
      </button>
    );
```